core/templates/dev/head/pages/exploration_editor/ExplorationEditor.js

```diff
@@ -205,6 +207,13 @@ oppia.controller('ExplorationEditor', [
 
         StateEditorTutorialFirstTimeService.init(
           data.show_state_editor_tutorial_on_load, $scope.explorationId);
+
+        if (ExplorationRightsService.isPublic()) {
+          // Don't make user wait on stats to begin editing the exploration.
```

Comment:
I'm not sure how this comment applies to the following line (I mean, how would you fetch stats alternatively anyway)?

Reply:
Hmm, well I was considering loading it alongside all of the other exploration data, since this looks like the only other backend callout made in this function.

Reply:
Oh, so is the point to ensure that it's at the end? If so, maybe say that.

(Basically ... what are devs supposed to try and preserve in the future?)

Reply:
Alright, I've updated it. How's that?

Reply:
Looks great, thanks!